### PR TITLE
Disable Dark Mode on recent versions of macOS.

### PR DESCRIPTION
**Pull request by sinetek**
### Problem ###
Recent macOS releases have introduced a so-called “Dark Mode” feature meant to give mac users an alternative theme for their operating system.
As of this time, the stylesheets we have in the Qt application do not work well with this change of colours.

In other words:
dmode wrong

### Root Cause ###
The root cause is a change in the Qt styling system on macOS, broadly speaking the problem was introduced by Apple Inc.

### Solution ###
The way I fix this is by just enforcing the known-good Aqua style.

In other words:
dmode right

### Unit Testing Results ###
Build on a relatively recent version of macOS, and see if: 

1)  The build is successful.
2)  The user succeeds in launching the application to the main window.
3)  The main window is visible and usable.

Versions to test :  10.14, 10.13.   The former introduced the problem, check for regression on later.

### CODE REVIEW

**Comment by sinetek** [comment]
Addressed a small compile issue on other platforms.
Ready for review now.

**Comment by sinetek** [comment]
Why is the travis not pulling in new files?

File: src/Makefile.qt.include [hunk 1]
```diff
@@ -171,7 +171,8 @@ QT_MOC_CPP = \
```
Comment (codeofalltrades) [inline src/Makefile.qt.include:171]: qt/macdarkmode.h needs to be added to the 
BITCOIN_QT_H
and 
QT_MOC_CPP
sections

Comment (sinetek) [reply src/Makefile.qt.include:171]: I'm not using any Qt classes, just regular QT_H is enough

**Comment by sinetek** [comment]
Added the header file.

**Comment by 4x13** [comment]
NACK This won't compile natively on OSX, this PR ides with the following error. Once PR's 723, 727, 757, 758, 762 and 766 are merged.

Screenshot from 2020-04-02 22-38-20

**Comment by sinetek** [comment]
Found a workaround.  Waiting on my 10.13 install to test.

**Comment by CaveSpectre11** [comment]
utACK https://github.com/Veil-Project/veil/commit/a9bffafec3fa670ffaf3174ac5e6f96461189498

**Comment by seanPhill** [comment]
Screen Shot 2020-07-05 at 11 08 25 pm
Colour scheme looks fine now, while Catalina Dark Mode is on.
